lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -3215,25 +3218,7 @@ sub add_subfigure_row_breaks {
       # reset bookkeeping on prior breaks, be they author-deposited or computed
       $current_width = 0; }
     else {
-      # TODO: A tricky bit to consider here is graphics sized to \columnwidth
-      # which isn't yet accurately emulated in latexml itself.
-      # so where two or more graphics may have been expected in the PDF,
-      # latexml would only allow one as the \columnwidth would be defaulted to \textwidth.
-      my $block_width = $block->getAttribute('width');
-      # If we have an author annotation of the width, it will appear as an XML attribute,
-      # and should take precedence over the underlying asset width of the image blob
-      if (!$block_width) {
-        if (my $options = $block->getAttribute('options')) {
-          if ($options =~ /width=([^ "]+)/) {
```

Comment:
Note that this regexp does not extract a valid width in general, particularly when there are multiple keywords.